core/CMakeLists.txt

```diff
@@ -134,13 +116,13 @@ if(DEFINED _LF_CLOCK_SYNC_ON)
   endif()
 endif()
 
-# Link with thread library, unless we are on the Zephyr platform.
-if(NOT DEFINED LF_SINGLE_THREADED OR DEFINED LF_TRACE)
-    if (NOT PLATFORM_ZEPHYR)
-        find_package(Threads REQUIRED)
-        target_link_libraries(reactor-c PUBLIC Threads::Threads)
-    endif()
-endif()
+
+# FIXME: May want these to be application dependent, hence passed as
```

Comment:
I realize this is a copy pasta, but should we turn this into a GitHub issue?

Reply:
I made it such that it is configurable by passing flags to CMake, e.g. `cmake -Bbuild -DINITAL_EVENT_QUEUE_SIZE=15`